pkgs/data/misc/dns-root-data/default.nix

```diff
@@ -4,7 +4,7 @@ let
 
   rootHints = fetchurl {
     url = "http://www.internic.net/domain/named.root";
```

Comment:
Probably something like that would be preferable in the future: https://github.com/Mic92/iana-etc